examples/hyperbolic/tree.json

```diff
@@ -1 +1,699 @@
\ No newline at end of file
+[
```

Comment:
This is not supposed to be pretty printed. It was generated by deriving_json somehow and should stay compact.